ssg/build_yaml.py

```diff
@@ -570,6 +570,52 @@ def from_yaml(yaml_file, env_yaml=None):
         rule.validate_references(yaml_file)
         return rule
 
+    def normalize(self, product):
+        self.make_items_product_specific(product)
+
+    def make_items_product_specific(self, product):
+        product_suffix = "@{0}".format(product)
+        to_set = dict(
+            identifiers=(self.identifiers, False),
+            references=(self.references, True),
+        )
+        for name, (dic, allow_overwrites) in to_set.items():
+            try:
+                new_items = self._make_items_product_specific(
+                    dic, product_suffix, allow_overwrites)
+            except ValueError as exc:
+                msg = (
+                    "Error processing {what} for rule '{rid}': {msg}"
+                    .format(what=name, rid=self.id_, msg=str(exc))
+                )
+                raise ValueError(msg)
+            dic.clear()
+            dic.update(new_items)
+
+    def _make_items_product_specific(self, items_dict, product_suffix, allow_overwrites=False):
```

Comment:
This is a nice rename, However, there already is `make_items_product_specific`. Please rename one of them